packages/third-parties/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@next/third-parties",
+  "version": "13.4.5-canary.8",
+  "repository": {
+    "url": "vercel/next.js",
+    "directory": "packages/third-parties"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "rm -rf dist && tsc -d -p tsconfig.json && node update-third-parties",
+    "prepublishOnly": "cd ../../ && turbo run build",
+    "dev": "tsc -d -w -p tsconfig.json",
+    "typescript": "tsec --noEmit -p tsconfig.json"
+  },
+  "dependencies": {
+    "third-party-capital": "1.0.17"
+  },
+  "devDependencies": {
+    "@vercel/ncc": "0.34.0"
+  },
+  "peerDependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
Seems `react-dom` is not being used? it's using "next/script", should "next" also be list as peer dependencies?

Reply:
Haven't seen `next` added as a dependency in other packages where we use next features. Eg `@next/font`
LMK if I should add next as a dependency still.

Reply:
ideally it should also be added into next/font as it's also using next/* internals, so I think we should remove react-dom here and add `"next": ">= 13"` as peer dependency

Reply:
Can we add "next" as peer dep?